### ISSUE TITLE
[pnpjs - graph - user] Unable to retrieve specific dimension for profile picture using pnpJS

### ISSUE THREAD

**Issue by MozzaikJD** [pnp/pnpjs#2850]
### What version of PnPjs library you are using

3.x

### Minor Version Number

3.20.1

### Target environment

Browser App (Hosted external to Microsoft 365 platform)

### Additional environment details

N/A

### Question/Request

Hello pnpJS community,

I am trying to retrieve a specific dimension for a user's profile picture using pnpJS, but I am unable to find any options to do so. Currently, I am using the following code to retrieve the profile picture:

Graphfi.using(CacheNever()).users.getById(userId).photo

This generates the following call: [https://graph.microsoft.com/v1.0/users/{user-id}/photo/$value](https://graph.microsoft.com/v1.0/users/%7Buser-id%7D/photo/$value)

However, I would like to be able to retrieve a specific dimension for the profile picture, such as 120x120, which can be achieved using the following endpoint in Graph: (photoS)

[https://graph.microsoft.com/v1.0/users/{user-id}/photos/120x120/$value](https://graph.microsoft.com/v1.0/users/%7Buser-id%7D/photos/120x120/$value)
[MS Documentation](https://learn.microsoft.com/en-us/graph/api/profilephoto-get?view=graph-rest-1.0&tabs=http#path-parameters)

I would appreciate any help or advice on how to retrieve a specific dimension for the profile picture using pnpJS.

Thank you!

**Comment by bcameron1231** [comment]
Unfortunately, we do not support this with our native methods. However you can make the call yourself using PnPjs. 

Check out this documentation on how to extend the library to call any endpoint you want.
https://pnp.github.io/pnpjs/concepts/calling-other-endpoints/

Here is a quick code sample.

```
 const user = graph.users.getById('beau@tenant.onmicrosoft.com');
 var photoValue = await graphGet(GraphQueryable(user, "photos/120x120/$value").using(BlobParse()));
 const blobUrl = URL.createObjectURL(photoValue);
 document.getElementById("photoElement").setAttribute('src', blobUrl);
```

**Comment by bcameron1231** [comment]
Hi. It was just released :) Version 3.21.0
